R/dplyr.r

```
@@ -130,6 +130,32 @@ query.bigquery <- function(con, sql, .vars) {
   BigQuery$new(con, sql(sql), .vars)
 }
 
+#' @export
+#' @importFrom dplyr db_save_query
+db_save_query.bigquery <- function(con, sql, name, temporary=TRUE, ...) {
```

given that we don't support `temporary = TRUE`, it seems like we should raise an error (or at least warn) if that value is provided?